deploy/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -56,7 +56,7 @@ metadata:
       ]
     capabilities: Deep Insights
     categories: Storage
-    containerImage: quay.io/ocs-dev/ocs-operator:4.15.0
+    containerImage: quay.io/ocs-dev/ocs-operator:latest
```

Comment:
why should this be `latest`?

Reply:
Because we're still trying to claim this is an upstream repo, and the `main` branch should by default build to `latest`, *not* the "latest release version". Our downstream pipelines always explicitly set this variable anyway, so that should be of no consequence.

Reply:
Sure, but this is an annotation. Do you need *this* for the odf-operator repo, but not the actual images in the Deployment spec?

Reply:
I would like to have the actual tag at both the places.

Reply:
That's the thing, this *is* the actual tag. There's a difference between the CSV version and the tag used for our container images. The CSV version needs to be a valid semantic version, the tag does not. And indeed, in the CSV itself all the container images use a default tag of `latest`. This is an annotation that's supposed to be advertising the container image of the operator, so why lie?

Reply:
you are correct. While building and pushing the images I export the version tag which creates and push the images with the version.

Reply:
So.... if that's the case, doing this would also keep this field up to date with whatever tag you're using at build time rather than having to manually update it at random intervals. Can I update this one string field to actually reflect the container image being referenced in the CSV Deployments?

Reply:
I'll not again, this is a *generated* change from the line changed in the csv-merger tool.

Reply:
yes lets have the actual container image version

Reply:
Thank you! Could I get an Approved review?